middleware/include.js

```diff
@@ -4,10 +4,11 @@ var path = require('path');
 var mm = require('micromatch');
 var typeOf = require('kind-of');
 var extend = require('extend-shallow');
+var isWindows = require('is-windows');
```

Comment:
Don't use a library for this. You can use `os.platform() === 'win32'` - https://nodejs.org/api/os.html#os_os_platform